### PR TITLE
add more algorithm constants

### PR DESCRIPTION
**Pull request by ezekielnewren**

### CODE REVIEW

**Comment by ezekielnewren** [comment]
These constants were copied from python's yubihsm[http]==2.1.0 library.

File: commands/types.go [hunk 1]
```diff
@@ -84,17 +84,51 @@ const (
...
 	AlgorithmYubicoAESAuthentication Algorithm = 38
-	AlgorithmAES128CCMWrap           Algorithm = 29
+	AlgorithmAES192YUBICOOTP         Algorithm = 39
+	AlgorithmAES256YUBICOOTP         Algorithm = 40
 	AlgorithmAES192CCMWrap           Algorithm = 41
 	AlgorithmAES256CCMWrap           Algorithm = 42
+	AlgorithmECECDSASHA256           Algorithm = 43
+	AlgorithmECECDSASHA384           Algorithm = 44
+	AlgorithmECECDSASHA512           Algorithm = 45
 	AlgorithmED25519                 Algorithm = 46
```
Comment (hendrikhofstadt) [inline commands/types.go:132]: Now that almost all are supported, let's also add secp224r1 (47). https://developers.yubico.com/YubiHSM2/Concepts/Algorithms.html

**Review by hendrikhofstadt — CHANGES_REQUESTED** [review]
Looks good to me, please add the 1 missing curve.

**Comment by hendrikhofstadt** [comment]
Appreciate your contribution.

**Pull request merged**